content/2019-01-12-Anniversary-meetup.md

```diff
@@ -0,0 +1,49 @@
+Title: PyDelhi Anniversary Meetup at Microsoft Gurugram
+Date: 2019-01-12
+Category: Meetup
+Slug: pydelhi-meetup-12-January-2019
+Author: Kaushlendra Pratap j
+Email: kaushlendrapratap.9837@gmail.com
+
+#PYDELHI BLOG
+
+##PYDELHI & BLOCKCHAIN Devs ANNIVERSARY MEETUP AT Microsoft Gurugram
+
+This time the meetup had something special to give. The 12/Jan/2019 meetup was not regular meetup it was a combined PyDelhi and Blockchain Devs anniversary meetup. The meetup was hosted in Microsoft Gurugram. Since This also was meetup was hosted in collaboration of four different communities i.e PyDelhi, BlockchainDevs, Open-Source-Networking-user-group This time around the meetup kicked off with the introduction of pydelhi and blockchain developer groups, the roadmap for the future and the past achievements were recalled where they elaborated about the future events. Demands of ideas for future events, suggestions and volunteers also raised there.
+
+![Anniversary-meetup](/images/12-01-19-Anniversary-meetup3.jpeg)
+
+Then talk on "Assistive tech for the blind in python" was given by Dinesh Kaushal [NVDA](https://github.com/nvaccess/nvda). where he mentioned the possible ways and solutions for the tech for visually impaired people, After which he presented the proposal for the people who were interested in contributing to the project.
+
+After the first few talks, we were asked to have a Break and Networking Session, which was really fun.
+
+
+After which Arif Khan gave a very informative talk on edge computing, where he discussed the future with edge computing and the differences between cloud computing and edge computing.This experience in the industry and an informative slide helps us to understand the topic lot more.
+
+![Anniversary-meetup](/images/12-01-19-Anniversary-meetup1.jpeg)
+
+
+After a quick coffee break, Vipin Rathi took a talk on the network technologies involved in edge computing the future scope of 5g technology and how it will affect the aspects of edge computing after which he also discussed the aspects of the blockchain.
```

Comment:
is it possible to link slides of the talk?

Reply:
I don't think they gave the link of slide.Let me ask, if they have one I will update 😊 @bhanuvrat

Reply:
@Quanon can you help us with the talk slides?

Reply:
Let's post without the slides, it can be updated later on if possible.